sway-lib-std/src/tx.sw

```diff
@@ -43,6 +43,7 @@ const TX_OUTPUTS_COUNT_OFFSET = 10304;
 const TX_WITNESSES_COUNT_OFFSET = 10312;
 const TX_RECEIPTS_ROOT_OFFSET = 10320;
 const TX_SCRIPT_START_OFFSET = 10352;
+const VM_ADDRESS_ZERO = 0;
```

Comment:
Maybe change this to `TX_ID_OFFSET = 0` following the format of the other consts? It is possible that the transaction ID may be stored at a different location in the future (though unlikely I suspect).

Reply:
Sure. 
A big change coming soon (TM) will be the change to using the `GTF` opcode rather than hardcoded offsets, but I will modify this for the time being.

Reply:
Oh nice.. I didn't know if `GTF` handles tx id or not but I guess it does!

Reply:
Now I'm second_guessing myself 😆 
But I will make this change regardless.

Reply:
![image](https://user-images.githubusercontent.com/17053076/177204191-c6c6e3da-d6c2-4073-9300-72945c151f9f.png)

Reply:
No that's not tx ID, that's the transaction ID part of the pointer for an input. `GTF` only handles tx fields, not the transaction ID. It could be added to `GM` though!